modern-phytomorphology.csl

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Modern Phytomorphology</title>
+    <title-short>ModPhytomorphol</title-short>
+    <id>http://www.zotero.org/styles/modern-phytomorphology</id>
+    <link href="http://www.zotero.org/styles/modern-phytomorphology" rel="self"/>
+    <link href="hhttps://phytomorphology.org/journal/author-guidelines/" rel="documentation"/>
+    <author>
+      <name>Andrew Novikoff</name>
+      <email>novikoffav@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issn>2226-3063</issn>
+    <eissn>2227-9555</eissn>
+    <updated>2016-10-01T04:26:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <names variable="editor translator" delimiter="; " prefix="(" suffix=")">
+      <name font-variant="small-caps" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
```

Comment:
Hi! I am very new in programming and csl, and therefore made so much errors. Thank you for helping me in improving the code!